src/Payfast.php

```diff
@@ -223,6 +244,11 @@ public function validSignature($signature)
 
     public function validateHost($request)
     {
+        // alow local testing
+        if (env('APP_ENV') !== 'production') {
+            return true;
+        }
```

Comment:
Hope you don't mind this, it makes testing with something like Ngrok much easier